aws/cluster.tf

```diff
@@ -198,6 +198,13 @@ resource "aws_iam_role_policy_attachment" "eks_node_group_3" {
   role       = aws_iam_role.eks_node_group.name
 }
 
+# New requirement to run the EBS CSI Driver
+# which is a new requirement to mount persistent volumes
+resource "aws_iam_role_policy_attachment" "eks_node_group_4" {
```

Comment:
Should this be added to [aws_eks_node_group.eks.depends_on](https://github.com/smallstep/run-anywhere-terraform/pull/21/files#diff-d554e3a2bea6ae5de1c9c715da5f78ffbc79ee41feea2449c9958ed3d6df2c36R103)?